docs/source/INSTRUCTORS.md

```diff
@@ -0,0 +1,91 @@
+
+
+### What are the duties of an instructor?
+
+- Familiarize themselves with the school practices and structure. 
+- Familiarize themselves with module content and assignments. 
+- Help students navigate the Discord course website and materials.
+- Be present during the project pitches and wrap-up presentations. 
+- Guide students with queries and give feedback on their projects.
+- Keep track of the student’s progress.
+- Report technical and practical problems to their local hub organizers.
+
+
+### What are the benefits of being an instructor?
+
+- Gaining practical and theoretical knowledge of the course materials.
+- Improving teaching and mentorship skills.
+- Peer working and collaborative research activities. 
+- Being part of a network of researchers and students from various institutes across the globe.
+
+
+### How many hours a day/week is an instructor expected to contribute? 
+
+An instructor is expected to be present both in-person & online for a minimum of 3 hours a day (1 hour for the morning, 1 hour for the afternoon Q&Q/Clinic sessions, and 1 extra hour to reply to any questions), totaling approximately 60 hours throughout the school. The instructor will be compensated for their time and contributions. 
+
+### How is an instructor associated with students?
+
+Students and instructors will be matched based on overlapping research interests and skill sets in a pod to meet regularly.
+
+
+### How is an instructor expected to communicate with the students?
+
+The communication across the day is run through an international Discord server. The instructors will encourage students to use the module/project-related channels to raise their questions and get feedback from their peers.
+
+Students will also have a chance to raise their questions during the Q&A hours when the instructor is expected to meet in person with the members of the pod they are assigned to.
+
+
+### Does an instructor have to answer the questions from students from other hubs?
+
+They are expected to but do not have to. Each hub will primarily be responsible for answering and helping its students. However, resolving some queries raised in the Discord server might address your expertise, or you might already know the answer. Then it would help the school community to resolve the issues quickly. 
+
+### How will an instructor follow the progress of a student?  
+
+For tracking the progress with students' completion of the module assignments, the local hub organizers that you are affiliated with will provide a means, mostly a shared sheet with the students' names and modules. So you will issue whether a student has completed an assignment based on their individual reporting.
+
+For the project improvements and assignments, meeting with the students 2 times a day will give you likely observations regarding how a student is progressing with their needs and executions.
```

Comment:
suggest replacing with "For project progress, regular meetings with students should give you a good idea of how they are doing."